src/main/java/com/modsen/pizzeria/mappers/CategoryMapper.java

```diff
@@ -5,6 +5,8 @@
 import com.modsen.pizzeria.dto.request.CreateCategoryRequest;
 import com.modsen.pizzeria.dto.request.UpdateCategoryRequest;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
 
 @Mapper(componentModel = "spring")

```

Comment:
можно сделать даже еще лучше
```suggestion
@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)

```